// airbnb-prop-types/build/index.js
var _and = require('./and');

var _and2 = _interopRequireDefault(_and);

var _childrenHavePropXorChildren = require('./childrenHavePropXorChildren');

var _childrenHavePropXorChildren2 = _interopRequireDefault(_childrenHavePropXorChildren);

var _childrenOfType = require('./childrenOfType');

var _childrenOfType2 = _interopRequireDefault(_childrenOfType);

var _componentWithName = require('./componentWithName');

var _componentWithName2 = _interopRequireDefault(_componentWithName);

var _forbidExtraProps = require('./forbidExtraProps');

var _forbidExtraProps2 = _interopRequireDefault(_forbidExtraProps);

var _mutuallyExclusiveProps = require('./mutuallyExclusiveProps');

var _mutuallyExclusiveProps2 = _interopRequireDefault(_mutuallyExclusiveProps);

var _mutuallyExclusiveTrueProps = require('./mutuallyExclusiveTrueProps');

var _mutuallyExclusiveTrueProps2 = _interopRequireDefault(_mutuallyExclusiveTrueProps);

var _nChildren = require('./nChildren');

var _nChildren2 = _interopRequireDefault(_nChildren);

var _nonNegativeInteger = require('./nonNegativeInteger');

var _nonNegativeInteger2 = _interopRequireDefault(_nonNegativeInteger);

var _or = require('./or');

var _or2 = _interopRequireDefault(_or);

var _range = require('./range');

var _range2 = _interopRequireDefault(_range);

var _restrictedProp = require('./restrictedProp');

var _restrictedProp2 = _interopRequireDefault(_restrictedProp);

var _uniqueArray = require('./uniqueArray');

var _uniqueArray2 = _interopRequireDefault(_uniqueArray);

var _uniqueArrayOf = require('./uniqueArrayOf');

var _uniqueArrayOf2 = _interopRequireDefault(_uniqueArrayOf);

var _withShape = require('./withShape');

var _withShape2 = _interopRequireDefault(_withShape);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }

module.exports = {
  and: _and2['default'],
  childrenHavePropXorChildren: _childrenHavePropXorChildren2['default'],
  childrenOfType: _childrenOfType2['default'],
  componentWithName: _componentWithName2['default'],
  forbidExtraProps: _forbidExtraProps2['default'],
  mutuallyExclusiveProps: _mutuallyExclusiveProps2['default'],
  mutuallyExclusiveTrueProps: _mutuallyExclusiveTrueProps2['default'],
  nChildren: _nChildren2['default'],
  nonNegativeInteger: _nonNegativeInteger2['default'],
  or: _or2['default'],
  range: _range2['default'],
  restrictedProp: _restrictedProp2['default'],
  uniqueArray: _uniqueArray2['default'],
  uniqueArrayOf: _uniqueArrayOf2['default'],
  withShape: _withShape2['default']
};
